src/Build/Logging/BinaryLogger/ProjectImportsCollector.cs

```diff
@@ -169,7 +195,7 @@ private static string CalculateArchivePath(string filePath)
             return archivePath;
         }
 
-        public void Close()
+        public void Close(bool closeStream = true)
```

Comment:
_Super_ nit:
```suggestion
        public void Close(bool closeStream = false)
```
Passing a parameter to enable something rather than disable something makes more sense in my opinion.